{
  "cve_id": "CVE-2018-6596",
  "github_data": {
    "repository": "anymail/django-anymail",
    "fix_commit": "c07998304b4a31df4c61deddcb03d3607a04691b",
    "related_commits": [
      "c07998304b4a31df4c61deddcb03d3607a04691b",
      "db586ede1fbb41dce21310ea28ae15a1cf1286c5",
      "c07998304b4a31df4c61deddcb03d3607a04691b",
      "db586ede1fbb41dce21310ea28ae15a1cf1286c5"
    ],
    "patch_url": null
  },
  "vulnerability_details": {
    "cvss_score": null,
    "cvss_vector": null,
    "cwe_id": "CWE-200",
    "description": "webhooks/base.py in Anymail (aka django-anymail) before 1.2.1 is prone to a timing attack vulnerability on the WEBHOOK_AUTHORIZATION secret, which allows remote attackers to post arbitrary e-mail tracking events.",
    "attack_vector": null,
    "attack_complexity": null
  },
  "temporal_data": {
    "published_date": "2018-02-03T21:29:00.210",
    "last_modified": "2024-11-21T04:10:58.043",
    "fix_date": null
  },
  "references": [
    {
      "url": "https://bugs.debian.org/889450",
      "source": "cve@mitre.org",
      "tags": [
        "Issue Tracking",
        "Patch",
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://github.com/anymail/django-anymail/commit/c07998304b4a31df4c61deddcb03d3607a04691b",
      "source": "cve@mitre.org",
      "tags": [
        "Patch"
      ]
    },
    {
      "url": "https://github.com/anymail/django-anymail/commit/db586ede1fbb41dce21310ea28ae15a1cf1286c5",
      "source": "cve@mitre.org",
      "tags": [
        "Patch"
      ]
    },
    {
      "url": "https://github.com/anymail/django-anymail/releases/tag/v1.2.1",
      "source": "cve@mitre.org",
      "tags": [
        "Release Notes"
      ]
    },
    {
      "url": "https://github.com/anymail/django-anymail/releases/tag/v1.3",
      "source": "cve@mitre.org",
      "tags": [
        "Release Notes"
      ]
    },
    {
      "url": "https://www.debian.org/security/2018/dsa-4107",
      "source": "cve@mitre.org",
      "tags": [
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://bugs.debian.org/889450",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Issue Tracking",
        "Patch",
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://github.com/anymail/django-anymail/commit/c07998304b4a31df4c61deddcb03d3607a04691b",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Patch"
      ]
    },
    {
      "url": "https://github.com/anymail/django-anymail/commit/db586ede1fbb41dce21310ea28ae15a1cf1286c5",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Patch"
      ]
    },
    {
      "url": "https://github.com/anymail/django-anymail/releases/tag/v1.2.1",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Release Notes"
      ]
    },
    {
      "url": "https://github.com/anymail/django-anymail/releases/tag/v1.3",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Release Notes"
      ]
    },
    {
      "url": "https://www.debian.org/security/2018/dsa-4107",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Third Party Advisory"
      ]
    }
  ],
  "collection_metadata": {
    "collected_at": "2025-01-11T22:59:15.394585",
    "processing_status": "raw"
  }
}